auto_submit/lib/service/graphql_queries.dart

```diff
@@ -0,0 +1,187 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:gql/ast.dart';
+import 'package:gql/language.dart' as lang;
+
+/// Provide a way to encapsulate the names variables that will be needed for each
```

Comment:
Nit: Provides. Let's keep all docs consistent.

Reply:
Cleaned up this description.